controllers/user.js

```diff
@@ -119,7 +130,23 @@ class Users {
     };
     const token = generateToken(tokenPayload);
 
+    const activationUrl = `${userActivationUrl}${user.id}`;
+    const html = `<h1 style=" text-align:justify";margin-left:50%;
+          padding:15px">
+          Welcome To LearnGround </h1><br>
+          <h3 style=" text-align:justify";margin-left:50%>
+          The Den Of Great Ideas
+          </h3>
+           <strong style=" text-align:justify";margin-left:50%>
+           Your Registration was sucessfull </strong><br>
+           <strong style=" text-align:justify";margin-left:50%>
+           Click <a href="${activationUrl}">Activate</a> to activate 
+           your account
+           </strong><br>`;
+
+    sendEmail({ email, subject, html });
```

Comment:
I thought you were going to add a subject for this one. `Welcome` maybe?

As it is, the default value is what gets passed here.

Reply:
ok...Welcome is great

Reply:
Do you mean like a default?, the subject is already initialize above

Reply:
Oh, great. My bad. Seen now.